### PR TITLE
Regression tests solution services changes

### PR DESCRIPTION
**Pull request by imranbashir01**

### CODE REVIEW

File: tests/NHSD.GPIT.BuyingCatalogue.E2ETests.Framework/Browsers/BrowserFactory.cs [hunk 1]
```diff
@@ -23,7 +23,8 @@ public BrowserFactory(string browser)
         private static IWebDriver GetLocalChromeDriver()
         {
             return new ChromeDriver(
-                Path.GetDirectoryName(AppDomain.CurrentDomain.BaseDirectory),
```
Comment (dssherlock) [inline tests/NHSD.GPIT.BuyingCatalogue.E2ETests.Framework/Browsers/BrowserFactory.cs:26]: Best to delete commented out code-it all exists in source control anyway

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/StepTwo/Quantity.cs [hunk 2]
```diff
@@ -47,15 +50,21 @@ public void EditQuantity(string catalogueItemName)
              nameof(OrderController.Order)).Should().BeTrue();
         }
 
-        public void AmendEditQuantity(string catalogueItemName)
+        public void AmendEditQuantity(string solutionid)
         {
-            CommonActions.ClickLinkElement(ReviewSolutionsObjects.EditCatalogueItemQuantiyLink(GetCatalogueItemID(catalogueItemName)));
```
Comment (dssherlock) [inline tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/StepTwo/Quantity.cs:52]: Typo!

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/StepTwo/SolutionSelection/SelectEditAssociatedService.cs [hunk 3]
```diff
@@ -85,32 +85,33 @@ public void EditAssociatedService(string solutionName, IEnumerable<string> newAs
...
                     {
                         foreach (var oldAssociatedService in oldAssociatedServices)
                         {
-                            CommonActions.ClickCheckboxByLabel(oldAssociatedService);
+                            CommonActions.ClickLinkElement(ReviewSolutionsObjects.RemoveSolutionService(oldAssociatedService));
+                            CommonActions.PageLoadedCorrectGetIndex(
+                            typeof(CatalogueSolutionsController),
+                            nameof(CatalogueSolutionsController.RemoveService)).Should().BeTrue();
+                            var removeservice = $"Yes, I confirm I want to remove {oldAssociatedService}";
+
```
Comment (dssherlock) [inline tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/StepTwo/SolutionSelection/SelectEditAssociatedService.cs:97]: Camel casing

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/OrderingPages.cs [hunk 4]
```diff
@@ -444,8 +450,9 @@ public void EditAdditionalService(string solutionName, IEnumerable<string> newAd
             {
                 foreach (var additionalService in newAdditionalServices)
                 {
-                    SelectEditAndConfirmAdditionalServicePrice.SelectAndConfirmPrice();
-                    Quantity.AddQuantity();
```
Comment (dssherlock) [inline tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/OrderingPages.cs:448]: This bit of code is used all over the place in this file (for the different Id types). Possibly worth pulling out into a function?

**Review by dssherlock — DISMISSED** [review]

File: tests/NHSD.GPIT.BuyingCatalogue.E2ETests.Framework/Browsers/BrowserFactory.cs (hunk 1 above)
Comment (imranbashir01) [reply tests/NHSD.GPIT.BuyingCatalogue.E2ETests.Framework/Browsers/BrowserFactory.cs:26]: Removed the commented code

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/StepTwo/Quantity.cs (hunk 2 above)
Comment (imranbashir01) [reply tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/StepTwo/Quantity.cs:52]: Fixed the typo

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/StepTwo/SolutionSelection/SelectEditAssociatedService.cs (hunk 3 above)
Comment (imranbashir01) [reply tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/StepTwo/SolutionSelection/SelectEditAssociatedService.cs:97]: Fixed the Camel casing

File: tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/OrderingPages.cs (hunk 4 above)
Comment (imranbashir01) [reply tests/NHSD.GPIT.BuyingCatalogue.RegressionTests/Pages/Ordering/OrderingPages.cs:448]: will fix it as part of the cleaning code after his pr has been merged

**Review by dssherlock — APPROVED** [review]

**Review by bjss-colin — APPROVED** [review]

**Pull request merged**